ts/features/idpay/wallet/screens/AvailableInitiativesList.tsx

```diff
@@ -77,29 +77,32 @@ export const IdPayInitiativeListScreen = (props: Props) => {
   }, [dispatch, idWallet, areInitiativesInError]);
 
   return (
-    <BaseScreenComponent
-      headerTitle={TypedI18n.t("idpay.wallet.initiativePairing.navigation")}
-      goBack={true}
-    >
-      <View style={IOStyles.horizontalContentPadding}>
-        <H1>{TypedI18n.t("idpay.wallet.initiativePairing.header")}</H1>
-        <VSpacer size={16} />
-        {maskedPan && (
-          <View style={IOStyles.row}>
-            <LogoPayment name={brandToLogoPaymentMap[brand]} />
-            <HSpacer size={8} />
-            <H4>•••• {maskedPan}</H4>
-          </View>
-        )}
-        <VSpacer size={16} />
-      </View>
-      <ScrollView style={IOStyles.horizontalContentPadding}>
-        <IDPayInitiativesList
-          initiatives={idpayInitiatives}
-          idWallet={idWallet}
-        />
-        <VSpacer size={24} />
+    <>
+      <BaseHeader
+        headerTitle={TypedI18n.t("idpay.wallet.initiativePairing.navigation")}
+        goBack={true}
+      />
+      <ScrollView>
+        <View style={IOStyles.horizontalContentPadding}>
+          <H1>{TypedI18n.t("idpay.wallet.initiativePairing.header")}</H1>
+          <VSpacer size={16} />
+          {maskedPan && (
+            <View style={IOStyles.row}>
+              <LogoPayment name={brandToLogoPaymentMap[brand]} />
+              <HSpacer size={8} />
+              <H4>•••• {maskedPan}</H4>
+            </View>
+          )}
+          <VSpacer size={16} />
+        </View>
+        <View style={IOStyles.horizontalContentPadding}>
+          <IDPayInitiativesList
+            initiatives={idpayInitiatives}
+            idWallet={idWallet}
+          />
+          <VSpacer size={24} />
+        </View>
```

Comment:
Why did you remove `BaseScreenComponent`? 

```suggestion
 <BaseScreenComponent
      headerTitle={TypedI18n.t("idpay.wallet.initiativePairing.navigation")}
      goBack={true}
    >
      <ScrollView style={styles.container}>
        <H1>{TypedI18n.t("idpay.wallet.initiativePairing.header")}</H1>
        <VSpacer size={16} />
        {maskedPan && (
          <View style={styles.card}>
            <LogoPayment name={brandToLogoPaymentMap[brand]} />
            <HSpacer size={8} />
            <H4>•••• {maskedPan}</H4>
          </View>
        )}
        <VSpacer size={16} />
        <IDPayInitiativesList
          initiatives={idpayInitiatives}
          idWallet={idWallet}
        />
      </ScrollView>
    </BaseScreenComponent>
```

With styles:

```
const styles = StyleSheet.create({
  card: {
    flex: 1,
    flexDirection: "row",
    paddingVertical: 8
  },
  container: {
    flex: 1,
    padding: customVariables.contentPadding
  }
});
```

Much simpler. What do you think?